Gopkg.toml

```diff
@@ -144,6 +139,23 @@ required = [
   unused-packages = false
   non-go = false
 
+# The dependencies below are required for go-containrregistry.
+[[constraint]]
+  name = "github.com/google/go-containerregistry"
+  # HEAD as of 2020-02-27
+  revision = "ba53fa10e72c7373ef8068e21c9f102f0c20a940"
+
+[[override]]
+  name = "github.com/vdemeester/k8s-pkg-credentialprovider"
```

Comment:
It is probably notable that this removes one of the blockers for Go module support

Reply:
Right, I'll add that to the description for more weight!